client/extensions/zoninator/components/settings/zones-dashboard/style.scss

```diff
@@ -12,11 +12,18 @@
 
 .zones-dashboard__zone-description {
 	color: $gray-dark;
+	height: 15px;
+	width: 56%;
+
+	small {
```

Comment:
Hmm, not sure we should be styling by tag selector here... Maybe have a designer weigh in on this?